infra/charts/feast-feature-server/templates/rbac.yml

```diff
@@ -0,0 +1,71 @@
+{{- if .Values.metrics }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: opentelemetry-targetallocator-sa
+  namespace: {{ .Values.namespace }}
```

Comment:
I think you need to update here to the namespace

Reply:
this line shouldn't be necessary ... helm should deploy to the appropriate namespace. the existing templates don't specify this line, so we should probably follow suit